src/components/CarouselRecommendationCard/CarouselRecommendationCard.fixture.tsx

```diff
@@ -14,11 +14,14 @@ const image: Picture = {
   title: 'product-basket',
 };
 
-const socialMedia: LinkProps[] = [
-  { href: 'https://t.me' },
-  { href: 'https://vk.com' },
-  { href: 'https://ok.ru' },
-];
+const button: ButtonWithIconProps = {
+  href: 'https://t.me',
+  target: '_blank',
+  version: 'grey',
+  rounded: false,
+};
+
+const socialMedia: ButtonWithIconProps[] = [button];
```

Comment:
```suggestion
const socialMedia: ButtonWithIconProps[] = [{
  href: 'https://t.me',
  target: '_blank',
  version: 'grey',
  rounded: false,
}];
```

вроде button самостоятельно не используется нигде